### PR TITLE
Adds several new clothing items and backpacks, changes hoodie sprites, and swaps out discount godzilla's clothing

### PR DESCRIPTION
**Pull request by DimWhat**
[WIKI] [SPRITES]

## About the PR 
Adds several sprites by Kobold Commando to the game including: 

- Several Hoodies and a resprite of the hooded and object version of hoodies 
- Dinosaur pajamas
- A reference to monkey island
- Two new hats
- Two new backpacks (With satchel versions!)

image

Monkey versions by me.
This PR also changes up the attire of the discount godzilla Halloween role to a much goofier set, lowers the price of the orange hoodie to match the blue one, and changes the name of the white pants with a red tie sprite for monkeys so that it actually works.
## Why's this needed? 
More clothing items is good, also the hoods on hoodies could use some work as well as the item versions of them. Also, i can't get over how silly discount godzilla would look like during halloween.
image

## Changelog

```changelog
(u)Kobold Commando & DimWhat
(*)Added several new clothing items to the clothing booth and two backpacks to Geoff Honkington.
(+)Changed the clothing of a certain lizardy halloween role
```

### CODE REVIEW

**Comment by BatElite** [comment]
Isn't the sock hat pretty much the pajama hat?

**Comment by DimWhat** [comment]
> Isn't the sock hat pretty much the pajama hat?

When making this I never compared the two sprites, yeah it's pretty much exactly the PJ hat. Kobold seemed to not know we even had a PJ hat in the game. I'll take it out.

**Comment by DimWhat** [comment]
oh crap pressed the wrong button

File: code/modules/vending/clothingbooth_items.dm [hunk 1]
```diff
@@ -863,3 +921,17 @@ ABSTRACT_TYPE(/datum/clothingbooth_item/costume)
...
+	slot = SLOT_W_UNIFORM
+	category = "Costumes"
+	cost = PAY_TRADESMAN/3
+
+/datum/clothingbooth_item/dinosuar
+	name = "Dinosaur Pajamas"
+	path = /obj/item/clothing/suit/gimmick/dinosaur
+	slot = SLOT_WEAR_SUIT
+	category = "Costumes"
+	cost = PAY_TRADESMAN/2
```
Comment (Sovexe) [inline code/modules/vending/clothingbooth_items.dm:937]: should be under the costume subtype

File: code/modules/vending/clothingbooth_items.dm [hunk 2]
```diff
@@ -656,6 +707,13 @@ ABSTRACT_TYPE(/datum/clothingbooth_item/head)
 	path = /obj/item/clothing/head/frog_hat
 	cost = PAY_TRADESMAN
 
+/datum/clothingbooth_item/link
+	name = "Hero Hat"
+	path = /obj/item/clothing/head/link
+	slot = SLOT_HEAD
+	category = "Headwear"
+	cost = PAY_TRADESMAN
```
Comment (Sovexe) [inline code/modules/vending/clothingbooth_items.dm:715]: should be under the head subtype

**Review by Sovexe — CHANGES_REQUESTED** [review]

**Review by Sovexe — APPROVED** [review]

**Pull request merged**